lib/streamlit/dataframe_util.py

```diff
@@ -75,6 +75,9 @@
 _SNOWPANDAS_INDEX_TYPE_STR: Final = (
     "snowflake.snowpark.modin.plugin.extensions.index.Index"
 )
+_POLARS_DATAFRAME: Final = "polars.dataframe.frame.DataFrame"
+_POLARS_SERIES: Final = "polars.series.series.Series"
+_POLARS_LAZYFRAME: Final = "polars.lazyframe.frame.LazyFrame"
```

Comment:
super nit: order alphabetically (`LazyFrame` before `Series`)

Reply:
Yep, I will reorder everything after in the last PR that is related to dataframe formats. Otherwise, merging development back into my big PR will get very messy.